src/cheri/riscv-hybrid-integration.adoc

```diff
@@ -208,77 +208,12 @@ In {cheri_int_mode_name}, the capability authorizing the memory access is <<ddc>
 [#zicsr-section-default]
 === Changes to Zicsr Instructions
 
-{cheri_base_ext_name} extends all RISC-V CSRs which hold addresses, to hold capabilities.
-Therefore, such registers are extended to CLEN-bits in {cheri_base_ext_name} and a `c` suffix is added when referring to the entire CSR instead of the only the address field.
+When in {cheri_int_mode_name}, there is a special rule for updating extended CSRs (e.g., <<utid>>/<<utidc>> or <<jvt>>/<<jvtc>>)
 
-WARNING: This assumes that the Zcmt CSR is extended to be a capability when the base architecture is {cheri_base_ext_name}.
+* Writing an extended CSR writes the address field (XLEN-bits) only, and the full CSR is updated using <<SCADDR>> semantics.
+* Reading an extended CSR reads the address field (XLEN-bits) only
```

Comment:
Nit picking: one of the bullet points has a "." at the other one does not. We should choose whatever is consistent with the rest of the spec.

Reply:
I guess that can be another PR to make it consistent, we also need to look at the rest of the ISA spec